src/components/dialogs/OpenDialog/index.js

```diff
@@ -39,6 +39,14 @@ class OpenDialog extends Component {
     this.props.view_store.toggleOpenDialog();
   }
 
+  @autobind onKeyDown(event) {
+    const code = event.nativeEvent.code;
```

Comment:
You can also use `event.key` which seems to have better cross browser support https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key.

Better not to rely on `nativeEvent` property in case in the future we decide to move to othre React implementations like Preact